libs/oef-base/include/oef-base/utils/Uri.hpp

```diff
@@ -59,12 +59,12 @@ class Uri
                    "\" port=", port, " path=\"", path, "\"");
   }
 
-  inline std::string GetSocketAddress() const
+  std::string GetSocketAddress() const
```

Comment:
Here and below, bodies should be moved to cpps then.

Reply:
The inline was redundant here. This is OEF code so I'd rather avoid merge conflict-prone changes like moving entire implementations to cpp files until after mainnet. The main focus of this PR is on ledger code.